recipes/arg_router/all/test_package/CMakeLists.txt

```diff
@@ -1,9 +1,8 @@
 cmake_minimum_required(VERSION 3.15)
+project(test_package LANGUAGES CXX)
 
-project(conan_test_project
-        LANGUAGES CXX)
+find_package(arg_router REQUIRED CONFIG)
 
-find_package(arg_router REQUIRED)
-
-add_executable(conan_test_project "main.cpp")
-target_link_libraries(conan_test_project PUBLIC arg_router::arg_router)
+add_executable(${PROJECT_NAME} test_package.cpp)
+target_link_libraries(${PROJECT_NAME} PRIVATE arg_router::arg_router)
+target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_17)
```

Comment:
Isn't this set by the toolchain/profile?

Reply:
The value is absent in Conan 1.x by defautl so it need to be set explicit for things to work in CCI :)